app/scripts/modules/core/src/pipeline/config/stages/monitoreddeploy/DeploymentMonitorExecutionDetails.tsx

```diff
@@ -108,17 +111,23 @@ export class DeploymentMonitorExecutionDetails extends React.Component<
             <dl className="dl-narrow dl-horizontal">
               <dt>Summary</dt>
               <dd>{getDeploymentMonitorSummary(stage)}</dd>
-              <dt className="sp-margin-l-bottom">Deploy %</dt>
-              <dd className="sp-margin-l-bottom">
-                <span>{stage.context.currentProgress}%</span>
-              </dd>
-              <dt>Details</dt>
-              <dd>{details || notProvidedFragment}</dd>
-              <dt className="sp-margin-l-bottom">Log</dt>
-              <dd className="sp-margin-l-bottom">{log ? <pre>{log}</pre> : notProvidedFragment}</dd>
-              <dt>More info</dt>
-              <dd>{additionalDetails ? '' : notProvidedFragment}</dd>
-              {additionalDetails}
+              {stage.hasNotStarted ? (
+                ''
+              ) : (
+                <>
+                  <dt className="sp-margin-l-bottom">Deploy %</dt>
+                  <dd className="sp-margin-l-bottom">
+                    <span>{stage.context.currentProgress}%</span>
+                  </dd>
+                  <dt>Details</dt>
+                  <dd>{details || notProvidedFragment}</dd>
+                  <dt className="sp-margin-l-bottom">Log</dt>
+                  <dd className="sp-margin-l-bottom">{log ? <pre>{log}</pre> : notProvidedFragment}</dd>
+                  <dt>More info</dt>
+                  <dd>{additionalDetails ? '' : notProvidedFragment}</dd>
```

Comment:
not sure what's up with this markup... do you need an empty `<dd>` here if `additionalDetails` exists? Or could you consolidate this line and the one after it into:
```typescript
{additionalDetails || (<dd>notProvidedFragment</dd>)}
```

Reply:
yes, it's needed, because it's sort of a table:
if there is no additionalDetails:
```
more info | <not provided>
```

if we have the additionalDetails:
```
more info | 
detailKey | detailText/Link
detailKey | detailText/Link
```

Reply:
ah i see, thanks. These `<dd>` / `<dt>` layouts get me confused.

Reply:
agree!